.github/CONTRIBUTING.md

```diff
@@ -142,7 +156,9 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If you have an idea you'd like to discuss or which requires additional testing consider opening it as a draft pull request. 
```

Comment:
nit: add a comma: "additional testing, consider"

Reply:
I don;t know how it is in english atm, but in my language it could even be a sub-sentence completely:
`* If you have an idea you'd like to discuss**,** or which requires additional testing**,** consider opening it as a draft pull request. `